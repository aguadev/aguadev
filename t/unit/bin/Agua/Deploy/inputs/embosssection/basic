  [-one]               typeone    No default description of abc and
                                  rest of ABC description
  [-two]               typetwo    [default cde] Description of cde and
                                  rest of CDE description

