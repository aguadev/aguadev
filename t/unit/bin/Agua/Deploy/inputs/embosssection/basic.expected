[
    {
        "argument":"-one",
		"paramtype":"typeone",
		"value":"",
		"name":"one",
		"discretion":"required",
		"description":"No default description of abc and rest of ABC description"
    }
    ,
    {
        "argument":"-two",
		"paramtype":"typetwo",
		"value":"default cde",
		"name":"two",
		"discretion":"required",
		"description":"Description of cde and rest of CDE description"
    }
]
